hyo2/soundspeed/formats/readers/sea_and_sun.py

```diff
@@ -15,6 +15,13 @@ class SeaAndSun(AbstractTextReader):
     Info: https://sea-sun-tech.com/technology.html
     """
 
+    # A dictionary to resolve probe type from probe type token
+    probe_dict = {
+        'CTD': Dicts.probe_types['SST-CTD'],
+        'CTP': Dicts.probe_types['SST-CTP'],   # deprecated
```

Comment:
Why are we adding a probe type with a deprecated comment?

Reply:
Good question.

From the most up to date documentation from Sea&Sun Technology, there are really just two possible probe types: CTD (a cabled probe) and CTM (a memory probe). This information can be parsed from the header of a TOB file in the section dedicated to the calibration coefficients. For example (without the actual coefficients): 

> 001  CTD54 001  P  Press  dbar
001  CTD54 002  N   Temp  degC
001  CTD54 003  N   Cond mS/c

However, because of the way the TOB files are customizable, I've come across data files in which the "CTP" probe type is used. For example:

> 001 CTP054 001  P  Press  dbar
001 CTP054 002  N   Temp  degC
001 CTP054 003  N   Cond mS/cm

The number 54 refers to the probe's S/N. So this is EXACTLY the same probe. There is therefore nothing fundamentally different between the CTD and CTP probe types. To be consistent, I would therefore want to stick to the two possible probe types: CTD and CTM. But, for backward compatibility, I was thinking of leaving CTP has a valid (but deprecated) probe type.

Any thoughts?

Reply:
I would remove the "deprecated" comment because it may be interpreted as no more used by SSM. My understanding is that SSM will be able to read CTP probe type.